### DISCUSSION TITLE
脚本添加的 configvar 变更后不会更新头文件

### DISCUSSION THREAD

**Discussion by RimuruChan** [xmake-io/xmake#4238] [Q&A]
![image](https://github.com/xmake-io/xmake/assets/42002296/d5ca5590-0a0c-4e76-9786-ff46a0ac7048)
如图，在添加了tag之后配置文件没有随之更新

**Comment by waruqi** [comment]
不是这么用的 https://xmake.io/#/zh-cn/manual/project_target?id=targetadd_configfiles

有内置的 git 和 version 变量可以用

**Reply by RimuruChan** [reply]
![image](https://github.com/xmake-io/xmake/assets/42002296/78fccfbf-ef2a-4584-a2a5-506eca4e5f24)
即使使用原生支持也不会更新。

**Reply by waruqi** [reply]
得执行 xmake f -c ，不是每次构建都要去更新的，没必要
